projects/components/src/load-async/wrapper/load-async-wrapper.scss

```diff
@@ -1,4 +0,0 @@
-:host {
```

Comment:
this seems like it could cause some issues. Are the loading icons centering in their target space?

Reply:
Fixed.